### PR TITLE
feature/FOUR-13852: Launchpad Setup - Create a modal for Launch Settings launchpad

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
Create a modal for Launch Settings launchpad
The change related to the ModalSaveVersion will updated from Engines Team

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13852

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by pmPaulis** [comment]
The build was requested with the ticket https://processmaker.atlassian.net/browse/SCO-22276

https://ci-1b4116ae5b.engk8s.processmaker.net/

File: resources/lang/en.json [hunk 1]
```diff
@@ -1975,7 +1995,6 @@
   "View All Screens": "View All Screens",
   "View All Scripts": "View All Scripts",
   "View All Notifications": "View All Notifications",
-  "View All Requests": "View All Requests",
```
Comment (fagubla) [inline resources/lang/en.json:1978]: don't remove

File: resources/js/components/shared/InputImageCarousel.vue
Comment (fagubla) [inline resources/js/components/shared/InputImageCarousel.vue:1]: move to the folder **processes-catalogue**

File: resources/js/components/shared/LaunchpadSettingsModal.vue
Comment (fagubla) [inline resources/js/components/shared/LaunchpadSettingsModal.vue:1]: move to the folder **processes-catalogue**

**Review by fagubla — APPROVED** [review]

**Pull request merged**